src/main/java/org/kitteh/irc/client/library/feature/twitch/TwitchSupport.java

```diff
@@ -26,31 +26,7 @@
 import org.checkerframework.checker.nullness.qual.NonNull;
 import org.kitteh.irc.client.library.Client;
 import org.kitteh.irc.client.library.defaults.listener.DefaultVersionListener;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.Badges;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.BanDuration;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.BanReason;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.Bits;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.BroadcasterLang;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.Color;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.DisplayName;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.EmoteSets;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.Emotes;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.Id;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.Mod;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.MsgId;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.MsgParamMonths;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.MsgParamSubPlan;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.MsgParamSubPlanName;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.R9k;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.RoomId;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.Slow;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.SubsOnly;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.Subscriber;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.SystemMsg;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.Turbo;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.User;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.UserId;
-import org.kitteh.irc.client.library.feature.twitch.messagetag.UserType;
+import org.kitteh.irc.client.library.feature.twitch.messagetag.*;
```

Comment:
Please revert this change - no `*` imports.

Reply:
We don't have a style guide..

Reply:
True, but I'll get around to doing that darnit! And yeah, please revert the import change. Or I can do it before I pull it to save you time! :)